ibexa/content/4.0.x-dev/encore/package.json

```diff
@@ -2,7 +2,7 @@
   "devDependencies": {
     "@hotwired/stimulus": "^3.0.0",
     "@symfony/stimulus-bridge": "^3.0.0",
-    "@symfony/webpack-encore": "^1.7.0",
+    "@symfony/webpack-encore": "<=1.7.0 || >1.8.0",
```

Comment:
Is this actually correct? Any version equal or lower than `1.7.0`, or higher than `1.8.0`?

This would include `1.6.x` and earlier?

Reply:
It will install up to 1.7.0 or 1.8.1+.
If other packages would require 1.6.x it would be possible to install 1.6.x instead of failing, but in our case, it will install 1.7.0.